### PR TITLE
Add PSR-12 code style standards

### PR DESCRIPTION
**Pull request by Yozhef**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,11 @@
+###> idea setting ###
+.idea/*
+###< idea setting ###
```
Comment (khaperets) [inline .gitignore]: You can ignore the .idea directory globally
https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000154070/comments/115000258150

**Review by khaperets — COMMENTED** [review]
It is necessary to remove the .idea from the .gitignore

**Review by khaperets — APPROVED** [review]

**Comment by khaperets** [comment]
Thanks for your PR!

File: .gitignore (hunk 1 above)
Comment (Yozhef) [reply .gitignore]: Thanks for the link =)

**Pull request merged**